bokehjs/src/coffee/models/graphs/static_layout_provider.coffee

```diff
@@ -0,0 +1,24 @@
+import {LayoutProvider} from "./layout_provider"
+import * as p from "../../core/properties"
+
+export class StaticLayoutProvider extends LayoutProvider
+  type: "StaticLayoutProvider"
+
+  get_node_locations: (graph_source) ->
+    [x, y] = [[], []]
+    for i in graph_source.nodes.data.index
+      x.push(@layout[i][0] ? null)
```

Comment:
This needs to be renamed to something less confusing with existing layout. `graph_layout` would suffice.